src/main/java/com/android/volley/toolbox/DiskBasedCacheUtility.java

```diff
@@ -136,4 +281,18 @@ static void writeHeaderList(List<Header> headers, OutputStream os) throws IOExce
         }
         return result;
     }
+
+    static int headerListSize(@Nullable List<Header> headers) throws IOException {
+        if (headers == null) {
+            return 4;
+        }
+        int bytes = 4;
+
+        for (Header header : headers) {
+            bytes += header.getName().getBytes("UTF-8").length;
```

Comment:
we use UTF-8 here and UTF8 elsewhere. Can we be consistent? Why do we check for UnsupportedEncodingException there and not here?

Reply:
Would it make sense to require newer api for these methods, then use StandardCharsets.UTF-8? It's a nio class, and never will have an UEE.

Reply:
UTF-8 is the default and is always available

Reply:
I was getting compilation errors unless I specified UTF-8

Reply:
I don't think we can use StandardCharsets here because this code is shared with the DiskBasedCache code path, which needs more backwards compatibility. Just want to make sure we're consistent with the naming.

Reply:
Okay, I just changed this back. They should all now be "UTF-8"